### DISCUSSION TITLE
Cant delete removed type from array

### DISCUSSION THREAD

**Discussion by lotarbo** [sanity-io/sanity#3770] [Sanity Studio Support] [answered]
I remove 'partLink' from object types in array field, and in studio i cant delete this item
![image](https://user-images.githubusercontent.com/1766890/196460538-1fe8b432-4c7a-4cfc-a671-036c14d88a63.png)

**Comment by lobunto** [comment]
Not a permanent solution but you could add the `partLink` object type back, then delete the item in the array (the one in your screenshot), and then delete the `partLink` object type again.

**Reply by lotarbo** [reply]
Thank you. I already do the same) But I just want to mark this ui problem in v3

**Comment by bjoerge** [comment]
Thanks for reporting! This will be fixed in the next release.

**Comment by bjoerge** [comment, marked as answer]
This is now fixed in latest `@dev-preview` release
